7_install_cert_manager/1_install_cert-manager.sh

```diff
@@ -0,0 +1,6 @@
+kubectl get pods --namespace cert-manager | awk '{print $2}' | grep -c 1/1 | grep 3 && INSTALLED=true
+
+if [ "$INSTALLED" != "true" ]; then
+  kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+fi
+
```

Comment:
Verify if after the installation the cert manager was successfully installed with a for loop and sleep (verify the jenkinsfile.deploy of intellij-frontend)